recipes/catkin/meta.yaml

```
@@ -0,0 +1,60 @@
+package:
```

Use variables to avoid mistakes when updating
```suggestion
{% set version = '0.7.17' %}
{% set ros_version = 'melodic' %}

package:
```

---

I forgot to mention that these recipes are auto-generated from a `rosdistro` file so ideally, the script should not make any mistakes now or in the future.